src/egon/data/datasets/pypsaeursec/__init__.py

```diff
@@ -743,6 +743,7 @@ def links_to_etrago(neighbor_links, scn="eGon100RE", extendable=True):
                     "efficiency4",
                     "lifetime",
                     "p_nom",
+                    # "p_min_pu",
```

Comment:
Could you remove the lines which are commented out. Here and in line 756. Or are they needed somehow?

Reply:
Sorry about that, I removed these lines.